python/ray/tests/test_client_builder.py

```diff
@@ -283,5 +284,98 @@ def test_address_resolution(call_ray_stop_only):
             del os.environ["RAY_ADDRESS"]
 
 
+def mock_connect(*args, **kwargs):
+    """
+    Force exit instead of actually attempting to connect
+    """
+    raise ConnectionError
```

Comment:
Could we maybe try to pass out a mock to actually have the connect succeed. I just want to make sure that this warning does not stop the connection.